include/utils/mqtt_abstraction_impl.hpp

```diff
@@ -20,7 +20,7 @@
 
 #include <utils/thread.hpp>
 
-constexpr std::size_t MQTT_BUF_SIZE = 500 * 1024;
+constexpr std::size_t MQTT_BUF_SIZE = std::size_t{500} * std::size_t{1024};
```

Comment:
doesn't
`constexpr auto MQTT_BUF_SIZE = std::size_t(500*1024};` also work?

Reply:
ah, yeah that works :+1: